ozaria/site/components/interactive/PageInteractive/draggableOrdering/index.vue

```diff
@@ -0,0 +1,127 @@
+<script>
+  import BaseInteractiveTitle from '../BaseInteractiveTitle'
+  import OrderingSlots from './OrderingSlots'
+
+  export default {
+    components: {
+      'base-interactive-title': BaseInteractiveTitle,
+      'ordering-slots': OrderingSlots
+    },
+
+    props: {
+      interactive: {
+        type: Object,
+        required: true
+      },
+
+      introLevelId: {
+        type: String,
+        required: true
+      },
+
+      interactiveSession: {
+        type: Object,
+        default: undefined
+      },
+
+      courseInstanceId: {
+        type: String,
+        default: undefined
+      }
+    },
+
+    data () {
+      return {
+        draggableGroup: Math.random().toString(),
```

Comment:
Didnt understand the purpose of draggableGroup

Reply:
To allow dragging between two column instances of the `vue-draggable` library, the two draggable columns must have the same draggableGroup name. Making them different will make it not possible to drag between the two columns.